CommandDotNet.Tests/FeatureTests/Prompting/PrompterTests.cs

```diff
@@ -36,13 +36,57 @@ public void InjectedPrompterCanPromptForValues()
                 });
         }
 
+        [Fact]
+        public void PromptForArguments()
+        {
+            new AppRunner<App>()
+                .UsePrompting()
+                .Verify(new Scenario
+                {
+                    When =
+                    {
+                        Args = "Greeting",
+                        OnPrompt = Respond.WithText("world")
+                    },
+                    Then =
+                    {
+                        OutputContainsTexts = new List<string> {"Hello world"}
```

Comment:
fyi: you can use object ctor syntax like `OutputContainsTexts = {"Hello world"}`

Reply:
I believe what you meant is called 'array initializer expression'. I tried using it as in the attached screenshot, but I get an error. Any thoughts on what I'm doing bad?
![image](https://user-images.githubusercontent.com/29201949/80860707-28fc3880-8c72-11ea-932d-2ba8791e80b5.png)

Reply:
It's not the same as [collection initializer](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/object-and-collection-initializers).  

What I'm referring to works only in the object initializer for collection properties and only when the collection properties are already instantiated.  The syntax `OutputContainsTexts = {"Hello world"}` only works within the `new Scenario` object initializer and only because `OutputContainsTexts` is assigned a `List<string>` in the field initializer.  `public List<string> OutputContainsTexts { get; set; } = new List<string>();`

Reply:
I've had a hard time find a good google query to get a better explanation.  [This post](https://marcin-chwedczuk.github.io/object-and-collection-initializers-in-csharp) covers it very briefly toward the bottom

Reply:
@drewburlingame thank you for the explanation, will read through the post you linked to.

Reply:
Basically, it's using collection initializer, but since the collection is already instantiated, the `new List<string>` part can be skipped and you can just use the `{...}` syntactic sugar for the initializer.